src/utils.js

```diff
@@ -0,0 +1,94 @@
+import { wordDictionary2 } from './letters/WordDictionary2';
+import { wordDictionary3 } from './letters/WordDictionary3';
+import { wordDictionary4 } from './letters/WordDictionary4';
+import { wordDictionary5 } from './letters/WordDictionary5';
+import { letterPoints } from './letters/LetterPoints';
+
+  // Look in correct dictionary to see if word exists
+  function validWord(word) {
+    let wordDictionArray = [wordDictionary2, wordDictionary3, wordDictionary4, wordDictionary5];
+    let wordDictionary = wordDictionArray[word.length - 2];
+    return !!wordDictionary.find(item => {
+      return item === word.toLowerCase();
+    });
+  }
+
+  // get words from board and put them into an array
+  function getWords(wordNo,squares) {
+    // the squares which contain 1's,2's,3's,..... etc. defines each word
+    let wordN;
+    let wordsList =[];
+    let result = '';
+    for (let i = 1; i < wordNo + 1; i++) {
+      wordN = [];
+      for (let j = 0; j < squares.length; j++) {
```

Comment:
I feel like this `for` can be replaced with a `reduce`...would have to think about it harder tho. Much harder without typescript

Reply:
Too complex for me.